worlds/sc2wol/docs/setup_en.md

```diff
@@ -15,37 +15,66 @@ Starcraft 2 AP Client launches a custom version of Starcraft 2 running modified
 Starcraft 2 is installed. On some computers particularly if you have OneDrive running this may  fail.  The following 
-directions may help you in this case if you are on Windows. 
+directions may help you in this case if you are on Windows.
 
-1. Navigate to '%userprofile%'.  Easiest way to do this is to hit Windows key+R type in %userprofile% and hit run or 
-type in %userprofile% in the navigation bar of your file explorer. 
+1. Navigate to '%userprofile%'.  Easiest way to do this is to hit Windows key+R type in `%userprofile%` and hit run or 
+type in `%userprofile%` in the navigation bar of your file explorer.
 2. If it does not exist create a folder in her named 'Documents'.
-3. Locate your 'My Documents' folder on your pc.  If you navigate to 'My PC' on the sidebar of file explorer should be a
+3. Locate your 'My Documents' folder on your PC.  If you navigate to 'My PC' on the sidebar of file explorer should be a
 link to this folder there labeled 'Documents'.
-4. Find a folder labeled 'Starcraft 2' and copy it.
-5. Paste this Starcraft 2 folder into the folder created or found in step 2.
+4. Find a folder labeled 'StarCraft II' and copy it.
+5. Paste this 'StarCraft II' folder into the folder created or found in step 2.
 
-These steps have been shown to work for some people for some people having issues with launching the game.  If you are 
-still having issues check out our [Discord](https://discord.com/invite/8Z65BR2) for help.
\ No newline at end of file
+These steps have been shown to work for some people for some people having issues with launching the game.  If you are
+still having issues check out our [Discord](https://discord.com/invite/8Z65BR2) for help.
+
+## Running in Linux
+
+To run StarCraft 2 through Archipelago in Linux, you will need to install the game using Wine then run the Linux build of the Archipelago client.
+
+Make sure you have StarCraft 2 installed using Wine and you have followed the [Installation Procedures](#installation-procedures) to add the Archipelago maps to the correct location. You will not need to copy the .dll files. If you're having trouble installing or running StarCraft 2 on Linux, I recommend using the Lutris installer.
+
+Copy the following into a .sh file, replacing the values of **WINE** and **SC2PATH** variables to the relevant locations, as well as updating the path to the Archipelago client, if it is not in the same folder as the script.
+
+```sh
+# Let the client know we're running SC2 in Wine
+export SC2PF=WineLinux
+export PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python
+
+# FIXME Replace with path to the version of Wine used to run SC2
+export WINE="/usr/bin/wine"
+
+# FIXME Replace with path to StarCraft II install folder
+export SC2PATH="/home/user/Games/starcraft-ii/drive_c/Program Files (x86)/StarCraft II/"
+
+# Start the Archipelago client
+./Archipelago_0.3.2_linux-x86_64.AppImage Starcraft2Client
```

Comment:
hm... we are at _0.3.3 now. Maybe `./Archipelago_*.AppImage Starcraft2Client`? Bash auto-completes a `*` to whatever is in there - that's at least common for some install instructions i've seen.
```suggestion
./Archipelago_*.AppImage Starcraft2Client
```

Reply:
Went a little overboard with the change in case the user has multiple versions of AppImages in the same directory. Makes the script a bit harder to read though, unfortunately.